dbtype/mssql/deploy_file.sh

```diff
@@ -2,9 +2,21 @@ function deploy_file() {
   _deploy_file="${1}"
   # ${2} is the name of log file, which is optional
 
-  deploy_output=$(${db_binary} -d ${dbname} ${server_flag}${port_flag} ${user_flag} ${password_flag} -h -1 -e -b -i "${fn_basedir}"/dbtype/"${dbtype}"/pre_deploy.sql "$_deploy_file" "${fn_basedir}"/dbtype/"$dbtype"/post_deploy.sql)
+  #create a temp file to hold the concatenate of the pre_deploy, deploy and post_deploy files
+
+  filename="$(echo ${_deploy_file} | awk '{print $NF}')"
+  
+  tmpfile="/tmp/tmp_${filename}"
+ 
+  echo "$(cat "${fn_basedir}"/dbtype/"${dbtype}"/pre_deploy.sql >> "${tmpfile}")"
+  echo "$(cat "${_deploy_file}" | sed "s/\$(DBNAME)/${dbname}/g" >> "${tmpfile}")"
+  echo "$(cat "${fn_basedir}"/dbtype/"${dbtype}"/post_deploy.sql >> "${tmpfile}")"
+
+  deploy_output=$(${db_binary} -d ${dbname} ${server_flag}${port_flag} ${user_flag} ${password_flag} -h -1 -e -b -i "${tmpfile}")
   rc=$?
 
+  echo "$(rm -f "${tmpfile}")"  
```

Comment:
we should also `unset tmpfile` after we remove the file to get rid of the variable lookup to the filename. Why are you doing this as an echo instead of just as a `rm -f "${tmpfile}"`?